packages/api/src/command/medical/cq-directory/rebuild-cq-directory.ts

```diff
@@ -0,0 +1,105 @@
+import { Carequality } from "@metriport/carequality-sdk/client/carequality";
+import { sleep } from "@metriport/core/util/sleep";
+import { QueryTypes, Sequelize } from "sequelize";
+import { Config } from "../../../shared/config";
+import { capture } from "../../../shared/notifications";
+import { bulkInsertCQDirectoryEntries } from "./create-cq-directory-entry";
+import { parseCQDirectoryEntries } from "./parse-cq-directory-entry";
+
+const BATCH_SIZE = 1000;
+const cqDirectoryEntryTemp = `cq_directory_entry_temp`;
+const cqDirectoryEntry = `cq_directory_entry`;
+const cqDirectoryEntryBackup = `cq_directory_entry_backup`;
+
+const sqlDBCreds = Config.getDBCreds();
+const dbCreds = JSON.parse(sqlDBCreds);
+
+const sequelize = new Sequelize(dbCreds.dbname, dbCreds.username, dbCreds.password, {
+  host: dbCreds.host,
+  port: dbCreds.port,
+  dialect: dbCreds.engine,
+});
+
+export const rebuildCQDirectory = async (failGracefully = false): Promise<void> => {
```

Comment:
I recommend we use actual `function` declaration instead of "arrow functions".

The main reason is the fact arrow functions are anonymous and don't show up on stack trace.

More why here: https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/

Reply:
Oh, I didn't realize that! I do rmb running into some issues where `this` behaves differently with these, but I thought they were otherwise the same. Thx!

Reply:
I did copy this structure from the existing similar functions btw 😅